Pipelines/Templates/tests.yaml

```diff
@@ -78,50 +45,49 @@ steps:
-        Write-Output '              End PlayMode Tests Logs                '
-        Write-Output '====================================================='
-      }
-      else {
-        Write-Output 'PlayMode Tests log missing!'
       }
-
-      if (Test-Path $resultFile) {
+      finally {
         Write-Output '====================================================='
-        Write-Output '            Begin PlayMode Tests Results             '
-        Write-Output '====================================================='
-
-        Get-Content $resultFile
-
-        Write-Output '====================================================='
-        Write-Output '             End PlayMode Tests Results              '
+        Write-Output '               PlayMode Tests Finished               '
         Write-Output '====================================================='
+        if (Test-Path $logFile) {
+          Write-Output '====================================================='
+          Write-Output '             Begin PlayMode Tests Logs               '
+          Write-Output '====================================================='
+          Get-Content $logFile
+          Write-Output '====================================================='
+          Write-Output '              End PlayMode Tests Logs                '
+          Write-Output '====================================================='
+        }
+        else {
+          Write-Output 'PlayMode Tests log missing!'
+        }
+        if (Test-Path $resultFile) {
+          Write-Output '====================================================='
+          Write-Output '            Begin PlayMode Tests Results             '
+          Write-Output '====================================================='
+          Get-Content $resultFile
+          Write-Output '====================================================='
+          Write-Output '             End PlayMode Tests Results              '
+          Write-Output '====================================================='
+        }
+        else {
+          Write-Output 'PlayMode Tests results missing!'
+        }
+      Write-Output $($proc.ExitCode)
       }
-      else {
-        Write-Output 'PlayMode Tests results missing!'
-      }
-
-      Write-Output "Exit code: $($proc.ExitCode)"
-    }
-  displayName: 'Run PlayMode tests'
 
-- task: PublishTestResults@2
-  displayName: 'Publish Test Results'
-  inputs:
-    testResultsFormat: NUnit
-    testResultsFiles: 'test*.xml'
-    searchFolder: $(Agent.TempDirectory)
-    failTaskOnFailedTests: true
+    displayName: 'Run PlayMode tests'
+  - task: CopyFiles@2
+    inputs:
+      SourceFolder: 'D:\a\_work\1\a'
+      Contents: '**'
+      TargetFolder: $(Build.ArtifactStagingDirectory)\Temp
+  - task: PublishTestResults@2
+    inputs:
+      testResultsFormat: NUnit
+      testResultsFiles: 'test-results-playmode.xml'
```

Comment:
It might be worth leaving the previous `'test*.xml'` value, in expectation of edit mode tests being added down the road.

Reply:
for the life of me, I couldn't get that test*.xml to pull any files. I know it should take any file named test*whatever.xml.

I'll try it again to see if I can get it working now